dist/OBS_debian/debian.control

```diff
@@ -2,23 +2,23 @@ Source: seamly2d
 Section: graphics
 Priority: optional
 Maintainer: Susan Spencer <susan.spencer@gmail.com>
-Build-Depends: debhelper (>= 8.0.0), 
-               qtbase5-dev (>= 5.2.0), 
-               libqt5svg5-dev (>= 5.2.0), 
-               g++ (>= 4.7.0), 
-               qt5-default (>= 5.2.0), 
-               qttools5-dev-tools (>= 5.2.0), 
+Build-Depends: debhelper (>= 8.0.0),
```

Comment:
Obviously, much of the info in debian.control is years out of date. But Launchpad doesn't support latest versions of Qt, so our Ubuntu builds are going away in favor of the all-linux-distros-compatible Appimage build.